python/cudf/cudf/core/frame.py

```diff
@@ -6301,6 +6301,7 @@ def truediv(self, other, axis, level=None, fill_value=None):
 
     # Alias for truediv
     div = truediv
+    divide = truediv
```

Comment:
This will likely need a docs update in series.rst & dataframe.rst

Reply:
Good point. The DataFrame binary operation function docs are actually quite out of date so I updated a number of them.